cmd/state/forward.go

```diff
@@ -29,7 +29,7 @@ const LatestVersion = "latest"
 
 type forwardFunc func() error
 
-func forwardFn(bindir string, args []string, out output.Outputer, pj *project.Project) (forwardFunc, error) {
+func forwardFn(cfg updater.Configurable, bindir string, args []string, out output.Outputer, pj *project.Project) (forwardFunc, error) {
```

Comment:
IIRC the forwarder isn't used by the auto updater, it's only used for locked versions. In which case we don't want it to care about tags.